pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.bf2</groupId>
+  <artifactId>cos-catalog-debezium</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+
+  <properties>
+    <debezium.schemabuilder.version>1.9.0-SNAPSHOT</debezium.schemabuilder.version>
+    <debezium.base.version>1.9.0.Alpha1</debezium.base.version>
+    <kafka.version>3.0.0</kafka.version>
+    <descriptor.output.directory>${project.basedir}/descriptors</descriptor.output.directory>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Not needed in presence of `release`.

Reply:
true. that's a leftover.

Reply:
Ah, now I see why this was needed. An old (pre 3.8.0) Maven Compiler Plugin was used. I added the 3.9.0 version now and it's working.